frontend/src/components/regions/UserActions/index.jsx

```diff
@@ -54,6 +91,19 @@ function UserActionsUnconnected({
     fetchUserBurndownStats({ userId });
   }, [fetchCardCompletions, fetchUserBurndownStats, userId]);
 
+  useEffect(() => {
+    fetchActivityLogEntries({
+      actorUser: userId,
+      page: 3,
```

Comment:
should we not start fetching from the first page?

Reply:
i forgot to change it back t 1, im on it.